micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/configs/UICustomizations.js

```
@@ -0,0 +1,272 @@
+      //first we can identify which column it belongs to then we can return relevant result
+      switch (key) {
+        case "SANDBOX_MODULE_NAME":
+          // return value?.split(".")?.[0];
+          return (
+            <span className="link">
+              <Link
+                to={`/${window.contextPath}/employee/workbench/mdms-search-v2?moduleName=${value?.split(".")?.[0]}&masterName=${
+                  value?.split(".")?.[1]
+                }`}
+              >
+                {t(Digit.Utils.workbench.getMDMSLabel(`SCHEMA_` + value))}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)